packages/flagship/src/types.ts

```diff
@@ -153,6 +153,7 @@ export interface IOSConfig {
 
 export interface PodsConfig {
   sources: string[];
+  newPods: string[];
```

Comment:
You should make this optional so it's backwards compatible, including adding a check in `packages/flagship/src/commands/init.ts` to see if it's undefined.

Reply:
Actually, you should make sources optional, too, in case there's ever a need to add new pods without adding sources.

Reply:
@Cauldrath PR updated, but in fact there's no need for a sources optional, we're already checking if sources exist inside the function in packages/flagship/src/lib/cocoapods.ts   **if (sources && sources.length > 0) {**

Reply:
@Cauldrath  Let me do the same with sources then, so we have the same convention, and squash the commits to 1

Reply:
The Typescript object should still allow setting it to undefined.

Reply:
@Cauldrath PR updated.